go.mod

```diff
@@ -146,20 +146,20 @@ require (
 	helm.sh/helm/v3 v3.9.0
 	k8s.io/api v0.26.1
 	k8s.io/apiextensions-apiserver v0.25.12
-	k8s.io/apimachinery v0.26.1
-	k8s.io/apiserver v0.25.12
+	k8s.io/apimachinery v0.28.3
+	k8s.io/apiserver v0.28.3
```

Comment:
Just to be sure - already 1.28, not 1.27? In v2.8, we still use 1.27...

Reply:
Good catch. I re-did the work in a less intrusive way.